src/components/partner/partner.repository.ts

```diff
@@ -57,6 +57,8 @@ export class PartnerRepository extends DtoRepository<
       active: input.active,
       startDate: input.startDate ?? CalendarDate.local(),
       address: input.address,
+      websiteUrl: input.websiteUrl ?? '',
+      socialUrl: input.socialUrl ?? '',
```

Comment:
```suggestion
      websiteUrl: input.websiteUrl,
      socialUrl: input.socialUrl,
```
Empty strings are _not_ values :)

We should however add a migration to retroactively add these props to DB.
See [add-type.migration.ts](https://github.com/SeedCompany/cord-api-v3/blob/f9f3d68245076443c620171686c0fe98169a60f0/src/components/organization/migrations/add-type.migration.ts)

Reply:
Believe it or not, I had a migration and then back tracked on that thought. Will re-add

Reply:
Oof